tests/functional/spec/features/submit_page/submit_page_collapsible_summary.spec.js

```diff
@@ -30,15 +30,15 @@ describe("Collapsible Summary", () => {
       expect($(SubmitPage.collapsibleSummary()).getText()).to.contain("Property Details");
       expect($(SubmitPage.collapsibleSummary()).getText()).to.contain("House Details");
 
-      expect($(SubmitPage.insuranceAddressQuestion()).isDisplayed()).to.be.false;
-      expect($(SubmitPage.numberOfPeopleQuestion()).isDisplayed()).to.be.false;
+      expect($(SubmitPage.insuranceAddressQuestion()).getText()).to.equal("");
+      expect($(SubmitPage.numberOfPeopleQuestion()).getText()).to.equal("");
```

Comment:
How come these have changed?. It shouldn be displayed?.

Reply:
Because there were some changes to the way collapsibles are marked up in the latest DS release `isDisplayed` stopped working correctly and was always evaluating to the contents of the collapsible as being always displayed even when the collapsible was closed. So moving to this fixed that and now the tests work correctly